samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -204,7 +200,7 @@ There are several containers to distinguish between the various outputs. We sugg
 
 ## Customizing the Ingestion Client
 
-By default, the ARM template uses the newest version of the Batch Ingestion Client which can be found in this repository. If a custom version should be used, the paths to the binaries inside the deployment template must be edited to point to a custom published version (by default, our binaries are: 
+By default, the ARM template uses the newest version of the Ingestion Client which can be found in this repository. If a custom version should be used, the paths to the binaries inside the deployment template must be edited to point to a custom published version (by default, our binaries are: 
 
 * https://mspublicstorage.blob.core.windows.net/transcription-enabled-storage/FetchTranscription.zip, 
```

Comment:
I was thinking that we maybe add the hash of the binary somewhere in there, in the docs and maybe even in the name of the file - so it's easier for users to check that the published code in this repo actually matches our precompiled functions

Reply:
ok?

Reply:
later maybe?